src/spec-lang/tc/semcheck.ts

```diff
@@ -424,3 +429,37 @@ export function scFunctionCall(
 
     throw new Error(`Internal error: NYI semcheck for ${expr.pp()}`);
 }
+
+/**
+ * For Any expression of form old(e1) in `forall(type t in range) e(t)`
+ * throw an error if the expression depends on t.
+ * @param expr
+ * @param ctx
+ * @param typeEnv
+ * @param semMap
```

Comment:
Params can be removed as there is no info specified for their description.

Reply:
Fixed in 2c86523